simulatingrisk/batch_run.py

```diff
@@ -47,46 +48,50 @@ def riskyfood_batch_run(args=None):
     save_results("riskyfood", results)
 
 
-def hawkdove_batch_run(args):
+def hawkdove_singlerisk_batch_run(args):
     # params are:
     # grid_size,
     # include_diagonals=True,
-    # risk_attitudes="variable", or single
     # agent_risk_level=None,
     # hawk_odds=0.5,
 
-    if args.risk_attitudes == "variable":
-        params = {
-            "grid_size": 20,
-            "risk_attitudes": "variable",
-        }
-        iterations = 5
-    elif args.risk_attitudes == "single":
-        params = {
-            "grid_size": 20,
-            "risk_attitudes": "single",
-            "agent_risk_level": [0, 1, 2, 3, 4, 5, 6, 7, 8],
-        }
-        iterations = 1
```

Comment:
It's cool that these params can be stored elsewhere via the subclassed models

Reply:
I'm not certain I need to store `risk_attitudes` anymore strictly speaking but there was one chart that used it to check how to display and I didn't refactor that out yet, so I kept it.